core/templatetags/otis_extras.py

```diff
@@ -46,6 +46,14 @@ def getprofile(user: User) -> Optional[UserProfile]:
         return None
 
 
+@register.filter(name="getconfig")
+def getconfig(user: User, config: str) -> bool:
+    try:
+        return getattr(UserProfile.objects.get(user=user), config)
+    except Exception:
+        return False
```

Comment:
in general i think `except Exception` is bad because it makes it hard to catch things

i think it should be like, try to get `profile = UserProfile.objects.get(user=user)`, and if this throws an error return False, else proceed as normal

Reply:
The issue is that I don't actually know what exceptions it likes to throw :skull:

Reply:
Should be fixed